docs/cross-account-resource-management.md

```diff
@@ -0,0 +1,56 @@
+# Cross account resource management (CARM)
+
+## The problem
+
+Today adminstrative users can manage AWS resources using ACK. However, these resources are created in the same AWS account as the Kubernetes cluster. This leads to a suboptimal experience for users in large organisations: creating resources in multiple AWS accounts requires ACK to be installed and managed in many kubernetes clusters. Considering ACK has dozens of service controllers, this means administrators must manage hundreds of AWS service controller pods across many kubernetes clusters and many AWS Accounts with scoped roles.
+
+With cross account resource management, a single kubernetes cluster with ACK installed can manage the resources of many different AWS accounts. Kubernetes administrators will not need to install and manage multiple clusters and ACK controllers, and AWS administrators will spend less effort managing permissions to access the services across accounts.
+
+## Proposed implementation
+
+The proposed solution will solve the problem of cross account management within a single kubernetes cluster. Given an AWS Owner account ID it will look up an IAM Role ARN and call STS:AssumeRole to pivot the AWS SDK calls in order to execute in a new AWS Account and role context.
```

Comment:
@A-Hilaly, quick question - how will this work if we are planning to use Pod IRSA setup?

Reply:
Thank you @nithu0115! I'm currently studying this case and will address it in my next commit.